### PR TITLE
Ticket 18 - TopBar & Context Provider

### PR DESCRIPTION
**Pull request by kipp0**
Now uses Context Provider

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -3,16 +3,34 @@ import { MuiThemeProvider } from "@material-ui/core";
...
+import UserContext from "./context/User"
+
 import LandingPage from "./pages/Landing";
+import TopBar from "./components/TopBar";
 
 import "./App.css";
 
+
+
+
```
Comment (sina-jamshidi) [inline client/src/App.js]: random empty space. You guys should discuss on Slack/standup about using the same prettier settings (also if you're not using Prettier definitely set it up).

File: client/src/components/TopBar.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+			onClick={toggleDrawer('top', false)}
+			onKeyDown={toggleDrawer('top', false)}
+		>
+			<List>
+				{routes.map((text, index) => (
+					<Link to={`/${text}`}>
+						<ListItem button key={text}>
+							<ListItemText primary={text} />
+						</ListItem>
+					</Link>
```
Comment (sina-jamshidi) [inline client/src/components/TopBar.js]: 👍

File: client/src/components/TopBar.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+				))}
+			</List>
+		</div>
+	);
+
+	return (
+		<div className={classes.root}>
```
Comment (sina-jamshidi) [inline client/src/components/TopBar.js]: Try to use Grids instead of divs to stay uniform across the app

File: client/src/pages/Landing.js [hunk 4]
```diff
@@ -1,71 +1,76 @@
+
 import React, { Component } from "react";
```
Comment (sina-jamshidi) [inline client/src/pages/Landing.js]: This page can be deleted

File: client/src/components/TopBar.js [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+	const [anchorEl, setAnchorEl] = useState(null);
+	const open = Boolean(anchorEl);
+
+	const routes = ['chef']
+	const [top, setTop] = useState({
+		top: false,
+	});
+	const handleAuth = (event) => {
+		setAuth(event.target.checked);
+	};
```
Comment (sina-jamshidi) [inline client/src/components/TopBar.js]: Are we just using this for debug right now?

File: client/src/components/TopBar.js [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+		setTop({ ...top, 'top': open });
+	};
+
+
+	const list = (anchor) => (
+		<div
+			className={clsx(classes.list, {
+				[classes.fullList]: anchor === 'top' || anchor === 'bottom',
+			})}
+			role="presentation"
```
Comment (sina-jamshidi) [inline client/src/components/TopBar.js:89]: Why is this role here?

**Review by sina-jamshidi — COMMENTED** [review]
This looks okay to me but there's some overlap with the previous PR.
When you address comments try to acknowledge them in some way (thumbs up reaction, comment) then click "resolve". This way we know what comments you have addressed in your PRs.
Also, try to give more descriptive PR names. It is not very clear what this PR is about  when looking at all the PRs.

File: client/src/components/TopBar.js (hunk 6 above)
Comment (kipp0) [reply client/src/components/TopBar.js:89]: It was for something i was testing out I'll remove it

File: client/src/components/TopBar.js (hunk 5 above)
Comment (kipp0) [reply client/src/components/TopBar.js]: I'm currently using it as a toggle for the user icon in the nav bar which will be hidden when the user is logged out. I figured eventually this function would be used to trigger the logout in a future task.

File: client/src/App.js (hunk 1 above)
Comment (kipp0) [reply client/src/App.js]: We did. I installed it. Is it suppose to remove spaces like that? I normally have more space between my imports. But i'll remove it

File: client/src/components/TopBar.js (hunk 5 above)
Comment (kipp0) [reply client/src/components/TopBar.js]: should remove the handleAuth function?

File: client/src/App.js (hunk 1 above)
Comment (kipp0) [reply client/src/App.js]: I've made another push to this branch called - PR:Solution with a description on the changes commit tag e6da962

File: client/src/pages/Landing.js (hunk 4 above)
Comment (kipp0) [reply client/src/pages/Landing.js]: Landing page has been deleted

File: client/src/components/TopBar.js (hunk 6 above)
Comment (kipp0) [reply client/src/components/TopBar.js:89]: removed

File: client/src/components/TopBar.js (hunk 5 above)
Comment (kipp0) [reply client/src/components/TopBar.js]: handleAuth is removed

**Pull request merged**